yatemplate.el

```diff
@@ -84,12 +85,14 @@ the ordering and regular expression parts.
 Note that this will be used as the SEPARATORS argument of
 `split-string', so be careful when setting this to a value that
 has special meaning in regular expressions."
-  :group 'yatemplate)
+  :group 'yatemplate
+  :type 'string)
 
 ;;;###autoload
 (defcustom yatemplate-ignored-files-regexp "README.md$"
   "Regular expression matching files that do not contain `yatemplate-separator', but will generate no warning."
-  :group 'yatemplate)
+  :group 'yatemplate
+  :type 'string)
```

Comment:
This could be `regexp` (from https://www.gnu.org/software/emacs/manual/html_node/elisp/Simple-Types.html#Simple-Types).

Feel free to force-push with this suggestion applied in the commit.